apps/sms/index.html

```diff
@@ -128,6 +129,7 @@
   </head>
 
   <body role="application" class="skin-comms theme-communications">
+    <iframe src="/views/shared/shim_host.html" class="shim-host" height="0" width="0"></iframe>
```

Comment:
Can you check it still works if we hide the iframe ? (display: hidden for the class, or hidden attribute)